recursive_art.py

```diff
@@ -1,74 +1,20 @@
-""" Function creates art based on randomly generated functions. Takes about 20 seconds to run. """
+""" Function creates art based on randomly generated functions."""
```

Comment:
Good implementation of lambda functions - this was a good idea for this MP5. They're a really awesome feature of python, I'd encourage you to continue thinking about how to effectively use them in the future!